app/src/main/java/com/example/habithelper/fragments/HomeFragment.java

```diff
@@ -127,9 +127,8 @@ public boolean onTouch(View v, @SuppressLint("ClickableViewAccessibility") Motio
             @Override
             public void onClick(View v) {
                 ParseUser.logOut();
-                ParseUser currentUser = ParseUser.getCurrentUser();
-                Intent i = new Intent(getContext(), LoginActivity.class);
-                startActivity(i);
+                currentUser = ParseUser.getCurrentUser();
```

Comment:
I'm guessing it would be null - I was mimicking the code we were told to use in the CodePath instagram assignment, but I never understood why we had to call that line. I'll ask the CodePath instructors on Wednesday!

Reply:
Sounds good, 
You can also try executing it and verify the return value on the debugger. If it is null, you may need to handle this value to avoid dereferencing null